web_action_conditionable/readme/CONFIGURE.rst

```diff
@@ -0,0 +1 @@
+Not needed.
```

Comment:
No need for this file then?

Reply:
@alexey-pelykh then Remove it :smile:

Reply:
well, I don't think I can :) It's @mreficent PR after all

Reply:
done